### PR TITLE
add results of regression tests research on netbsd

### PR DESCRIPTION
**Pull request by PrzemyslawBanasiak13**
Signed-off-by: pbanasiak <przemyslaw.banasiak@3mdeb.com>

### CODE REVIEW

**Comment by PrzemyslawBanasiak13** [comment]
@SergiiDmytruk could you take a look at it?

**Comment by SergiiDmytruk** [comment]
I haven't looked into NetBSD that much, better to ask @Silentsky0.

**Comment by pcichowski** [comment]
LGTM overall. As for the `OS installation in automated test` point, I looked for some implementations of an unattended installer, but unfortunately no significant results came up.

File: netbsd/netbsd_regression_Q&A.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+1. Otherwise use your own discretion.
+
+- [Link](https://github.com/NetBSD/src/blob/trunk/tests/README)
+
+1. It would be more convenient to create new tests similarly to ones currently
+existing in repo. Introducing whole robot framework environment to the project
+may raise many questions and create many issues.
+
+## OS installation in automated test
```
Comment (macpijan) [inline netbsd/netbsd_regression_Q&A.md]: I can confirm that we do not need OS installation tests. We do not want to test OS installation at all here.

We just want to develop a set of regression tests which test our added functionalities to be included in the regression suite of the given OS. We will not use our typical test infrastructure at all here.

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**